credential-verification-service/src/api_types.rs

```diff
@@ -23,3 +33,39 @@ pub struct CreateVerificationRequest {
     // that is submitted on-chain.
     // pub public_info: HashMap<String, SerdeCborValue>,
 }
+
+/// API request payload for verifying a presentation
+/// endpoint: `/verifiable-presentations/verify`.
+#[derive(serde::Deserialize, serde::Serialize, Debug, Clone)]
+pub struct VerifyPresentationRequest {
+    /// Audit record id that the client wants to set for this presentation verification
```

Comment:
The `audit_record_id` is not really used in the verification of the presentation. Suggestion:

```suggestion
    /// Audit record id that the client wants to include in the audit anchor.
```